lib/awful/menu.lua

```diff
@@ -239,17 +241,37 @@ local function check_access_key(self, key)
    end
 end
 
+function menu:select_next(direction, current)
```

Comment:
It looks like an internal function we don't want to expose. 

Change the declaration to a local function (the signature will also need to be changed to receive the menu instance).